tests/test_batch.py

```diff
@@ -346,7 +347,7 @@ def test_get_job_error(
         assert (
             mock_request_exception.last_request.url
             == f"{self.sdk._client.endpoints.core}"
-            f"/api/v1/jobs/{job.id}"
+            f"/api/v2/jobs/{job.id}"
```

Comment:
So are we considering V1 deprecated now? Do we no longer want to support testing its behaviors?

Reply:
The v1 endpoints are going to be tested in our e2es which target older versions of the SDK.

The responsibility of the unit test is only to test the features of the SDK, which means only the endpoint the SDK uses. That's my understanding at least.

On a tangent note, I think the v1 endpoint is indeed considered deprecated now

Reply:
Not sure why that would be a tangent note, it was the first question I wanted an answer for.

Reply:
Yeah I rewrote that paragraph and messed up oops. 

Anyway the point was I don't think v1 endpoints should be tested here, do you disagree?